middlewares/studentAssessmentAccess.ejs

```diff
@@ -32,13 +32,13 @@
                       <div class="col-md-3 col-sm-6">
                         <%- include('../pages/partials/scorebar', {score: assessment_instance.score_perc}) %>
                       </div>
-                      <div class="col-md-6 col-sm-12 text-right">
-                          Assessment is <strong>closed</strong> and is no longer available
+                      <div class="col-md-6 col-sm-12 text-right assessment-closed-message">
+                          Assessment is <strong>closed</strong> and is no longer available.
                       </div>
                     </div>
 
                     <% if (authz_data.mode == 'SEB') { %>
-                        <p><a href="/pl/SEBquit">Click here to quit the Safe Exam Browser</a>
+                        <p><a href="/pl/SEBquit">Click here to quit the Safe Exam Browser</a></p>
```

Comment:
nice catch!

Reply:
All credit for this goes to @nwalters512 !